lib/source.js

```diff
@@ -146,17 +212,19 @@ function originalEndPositionFor (sourceMap, line, column) {
     source: beforeEndMapping.source,
     line: beforeEndMapping.line,
     column: beforeEndMapping.column + 1,
-    bias: LEAST_UPPER_BOUND
+    bias: GREATEST_LOWER_BOUND
```

Comment:
This logic has been thoroughly tested in c8, with several real world generators ([take a look at the tests here](https://github.com/bcoe/c8/blob/master/test/integration.js#L220)).

I don't think we want to fiddle with this logic, as it grows out of a lot of testing.